creator-node/src/services/sync/primarySyncFromSecondary.js

```diff
@@ -178,6 +179,16 @@ async function _primarySyncFromSecondary({
         }
       }
 
+      // Recompute storage paths to use this node's path prefix
+      fetchedCNodeUser.files.forEach((file) => {
```

Comment:
dont you have to re-assign this?

Reply:
map returns a new array but foreach updates in place, i think that's the difference between the two, but we should definitely test to confirm

Reply:
fair - maybe worth doing a local test for primarySync similar to secondarySync which will be the safest way to confirm?

Reply:
exactly, yeah, I used forEach here to do it in place. mad dog is failing so I'll retest anyway if re-running it doesn't work

Reply:
I should do a local test anyway, yeah